src/puppetlabs/puppetdb/cli/import.clj

```diff
@@ -96,7 +96,7 @@
                              options)
         construct-base-url (fn [{:keys [host port] :as options}]
                              (-> options
-                                 (assoc :base-url (utils/pdb-admin-base-url host port export/admin-api-version))
+                                 (assoc :base-url (utils/pdb-admin-base-url host port :v1))
```

Comment:
any reason not to keep the version vars we had previously? It'll eliminate the need to update multiple places down the road.